src/test/groovy/com/calgaryscientific/gradle/VeracodeMitigationInfoTest.groovy

```diff
@@ -63,4 +66,32 @@ class VeracodeMitigationInfoTest extends TestCommonSetup {
         assert is.readLine() == ''
         assert is.readLine() == 'action: appdesign'
     }
+
+    def 'Test printMitigationInfoErrors'() {
```

Comment:
it would be nice if the test name can indicate what business rule/behavior it is trying to test

Reply:
Just testing the one printing method... 
The business rule is getting tested here: https://github.com/calgaryscientific/veracode-gradle-plugin/pull/44/files#diff-fda0126fb095a7f2481f9af9f428cad7R97